gpiocontrol/resources/projects.xml

```diff
@@ -222,7 +222,7 @@ For more information, see <a href="http://kaaproject.github.io/kaa/docs/v0.10.0/
         <sdkLibDir>gpiocontrol/libs</sdkLibDir>
         <destBinaryFile></destBinaryFile>
     </project>
-    <bundle id="gpio_control">
+    <bundle id="gpiocontrol_demo">
```

Comment:
This should not be fixed by a roll-back of the bundle Id and naming. Instread, please move the resources bundle directory together with the icon to fix this, please.